### ISSUE TITLE
certain subcommands don't work

### ISSUE THREAD

**Issue by endzyme** [reactiveops/reckoner#29]
Might suggest removing them as they do not work today.

`reckoner generate` Stacktrace -- see below:
```
reckoner generate
2019-01-04 14:23:03 xps13-9370 root[16862] INFO Generating example course as course.yml
Traceback (most recent call last):
  File "/home/user/.pentagon/venv/bin/reckoner", line 11, in <module>
    sys.exit(cli())
  File "/home/user/.pentagon/venv/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/user/.pentagon/venv/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/user/.pentagon/venv/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/user/.pentagon/venv/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/user/.pentagon/venv/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/user/.pentagon/venv/local/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/user/.pentagon/venv/local/lib/python2.7/site-packages/reckoner/cli.py", line 62, in generate
    src = pkg_resources.resource_string("reckoner", "example-course.yml")
  File "/home/user/.pentagon/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1152, in resource_string
    self, resource_name
  File "/home/user/.pentagon/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1396, in get_resource_string
    return self._get(self._fn(self.module_path, resource_name))
  File "/home/user/.pentagon/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1519, in _get
    with open(path, 'rb') as stream:
IOError: [Errno 2] No such file or directory: '/home/user/.pentagon/venv/local/lib/python2.7/site-packages/reckoner/example-course.yml'
```

`reckoner version`
```
Traceback (most recent call last):
  File "/home/nhuanca/.pentagon/venv/bin/reckoner", line 11, in <module>
    sys.exit(cli())
  File "/path/to/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/path/to/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/path/to/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/path/to/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/path/to/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/path/to/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/path/to/reckoner/cli.py", line 72, in version
    print(reckoner.__version__)
NameError: global name 'reckoner' is not defined
```

**Comment by endzyme** [comment]
#56

**Comment by endzyme** [comment]
fixed in 1.0.0
